### ISSUE TITLE
general tab / dashboard missing from Overview

### ISSUE THREAD

**Issue by humm3r1** [wazuh/wazuh-kibana-app#1432]
I seem to not have a ton of tabs in either my custom install or in the pre-made OVA. Were they removed for some reason, or am I missing the option to turn them back on?

I have none of the tabs at the top (General, File Integrity, Policy monitoring, etc.) from here: 
https://lh6.googleusercontent.com/proxy/W4y_F5KFNqVTF8A7y5QUA3CUgW1fJOkPjViz80Ax_lsvHOnPSo7WKv154NV-Fa43SIDPoKP9AOJINQI8BYZfMhx7q7DCB6ofXSPqPPFzSl1f7vm2CzioZg3d-gyXAq3gReykkt-LMyVEP1A5UcML3r2nF6DjKek=w5000-h5000

This is what I see under overview (note the lack of tabs, not even a General tab now): https://i.ibb.co/rZmQwTv/Wazuh-Lack-of-Dashboards.jpg

**Comment by humm3r1** [comment]
I tried a bunch of different copies of the OVA/OVF files and had to roll back all the way down to wazuh3.0.0_6.1.0 before I had those tabs show up. I'm going to try upgrading now and see if they stick.

Was there a reason they got removed between 3.0.0 and now?

**Comment by juankaromo** [comment]
Hi @humm3r1 

The design works differently from several versions. Some changes were introduced to avoid forcing a heavy load of elastic on each load of the welcome page. This was solved by introducing extensions, which allow you to decide which modules you want to load at the beginning and which ones you do not. Despite this, we are undoubtedly trying to improve navigability day by day, and for example we have opened this issue https://github.com/wazuh/wazuh-kibana-app/issues/1323 to speed up navigation and depth levels and make the experience more comfortable.

So having said that, any suggestion is welcome, and of course we will study it.

Greetings.
